docs/blueprints/EP031.rst

```diff
@@ -249,7 +249,8 @@ V. Events
 ==========
 
   1. Listening
-    1. *kytos/mef_eline.(removed|deployed)*
+    1. *kytos/mef_eline.(removed|deployed|undeployed)*
+    2. *kytos.topology.updated*
```

Comment:
This is for keeping track of `proxy_interface` status changes, and also `proxy_port` metadata changes